resources/views/layouts/app.blade.php

```diff
@@ -32,19 +32,20 @@
 
                 <div class="collapse navbar-collapse" id="navbarSupportedContent">
                     <!-- Left Side Of Navbar -->
-                    <ul class="navbar-nav mr-auto">
-                        <li><a class="navbar-brand" href="{{ route('pages.index') }}"> Pages </a> </li>
-                        <li><a class="navbar-brand" href="{{ route('menus.index') }}"> Menus </a></li>
-                    </ul>
-
+                    @if (Auth::check())
+                        <ul class="navbar-nav mr-auto">
+                            <li><a class="navbar-brand" href="{{ route('pages.index') }}"> Pages </a> </li>
```

Comment:
Try to not have unnecessary spaces on each side of your text.

For instance, here you have `<a> Pages </a>`.
Just do this instead: `<a>Pages</a>`